### PR TITLE
feat: displayName in the textile explorer, reordered columns

### PR DESCRIPTION
**Pull request by ccomb**
[Story](https://www.notion.so/4f0841edbc29408eb147558e53fe2acd?v=40d0d544eb8e4c03891da1f9aad8b6cd&p=8001e835e8024b45a7014c4cb2e5b9e3&pm=s)

ecobalyse-private: textile_explorer

### CODE REVIEW

File: src/Data/Textile/Process.elm [hunk 1]
```diff
@@ -106,6 +109,11 @@ decode impactsDecoder =
         |> Pipe.required "alias" (Decode.maybe decodeAlias)
 
 
+getDisplayName : Process -> String
+getDisplayName process =
+    Maybe.withDefault process.name process.displayName
```
Comment (n1k0) [inline src/Data/Textile/Process.elm:114]: So this is the same fn as the other for Food. Would it be possible to just have one generic, like:

```elm
getDisplayName : { displayName : Maybe String, name : String } -> String
getDisplayName { displayName, name } =
    Maybe.withDefault name displayName
```

File: src/Page/Explore/TextileProcesses.elm [hunk 2]
```diff
@@ -2,7 +2,7 @@ module Page.Explore.TextileProcesses exposing (table)
 
 import Data.Dataset as Dataset
 import Data.Scope exposing (Scope)
-import Data.Textile.Process as Process exposing (Process)
+import Data.Textile.Process as Process exposing (Process, getDisplayName)
```
Comment (n1k0) [inline src/Page/Explore/TextileProcesses.elm:5]: Same

File: data/README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ d'abord un `make clean_data` (qui supprime le volume docker).
 - `make import_method` : pour importer EF 3.1 adapted dans Brightway.
   Assurez-vous d'avoir le bon fichier de données dans `data/`
 - `make export_food` : pour exporter les json pour le builder alimentaire
-- `make delete_database DB=<dbname>` : pour supprimer une base de données
+- `make delete_database DB=<dbname>` : pour supprimer une base de données (Ex avec espace: make delete_database DB="Ecoinvent\ 3.9.1")
```
[the remark this answers is not in the document]
Comment (ccomb) [reply data/README.md:30]: Sorry I can't help doing 1-line improvements while doing some work, I find it's easier and faster to bring them along with the main PR :-/

File: data/notebooks/explore.py [hunk 4]
```diff
@@ -411,7 +411,7 @@ def display_main_data(method, impact_category, activity):
             name = ", ".join(m[1:])
             scores[name] = {
                 "Indicateur": name,
-                "Amount": lca.score,
+                "Score": lca.score,
```
[the remark this answers is not in the document]
Comment (ccomb) [reply data/notebooks/explore.py:414]: This is a real fix so I'll do a separate PR for this

File: src/Data/Food/Process.elm [hunk 5]
```diff
@@ -306,12 +306,7 @@ encodeStringUnit unit =
 
 getDisplayName : Process -> String
 getDisplayName process =
-    case process.displayName of
-        Just displayName ->
-            displayName
-
-        Nothing ->
-            nameToString process.name
+    Maybe.withDefault (nameToString process.name) process.displayName
```
[the remark this answers is not in the document]
Comment (ccomb) [reply src/Data/Food/Process.elm]: :heavy_check_mark:

File: src/Data/Textile/Process.elm (hunk 1 above)
Comment (ccomb) [reply src/Data/Textile/Process.elm]: Actually I wouldn't have defined a function just for a withDefault.  I would rather remove it. The generic one looks more complicated for something clone to an identity: `getDisplayName {a, b} = withDefault a b`

Comment (n1k0) [reply src/Data/Textile/Process.elm]: Yeah so please just use Maybe.withDefault instad of dedicated functions doing the same, thanks.

Comment (ccomb) [reply src/Data/Textile/Process.elm]: Sorry I saw this after the merge. Will do another PR

Comment (n1k0) [reply src/Data/Textile/Process.elm]: @ccomb no it's fine, leave it like that

**Pull request merged**